portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,17 +37,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  String comNum =  "5";
```

Comment:
It may be good to explicitly use a "private" annotation for such class member variables.